charts/values.yaml.tpl

```diff
@@ -151,35 +151,23 @@ backingServices:
     secretName: rhtap-argocd-integration
     ingressDomain: {{ $ingressDomain }}
 
-# integrations:
```

Comment:
Please keep the `integrations`  for the moment. You can remove the `developerHub` and `catalogUrl` fields though.

Reply:
Sure, I've reverted this chunk. Please consider.